docs/workflows/index.md

```diff
@@ -4,4 +4,5 @@
 
 ## Code-based workflows
 
-- [Create an Airtable remote data block with code](workflows/airtable-with-code.md)
+- [Create an Airtable integration with code](workflows/airtable-with-code.md)
+- [Create a Zip Code integration template with code](workflows/zip-code-integration-template.md)
```

Comment:
```suggestion
- [Create a Zip Code integration template with code](workflows/zip-code-integration-template.md)
- Advanced usage: [Create an Airtable integration with code](workflows/airtable-with-code.md)
```

Reply:
Something like this is what I was thinking... thoughts?

I'm open to the idea that we shouldn't frame anything as Standard / Advanced right now since we don't really know the customer usage yet... but that's what I think of them as.

Reply:
I like it. Also totally fine 👍 🚢  with any docs PR, we will go through plenty of revisions and reorgs later, just need words at this point